cli/cdsctl/shell.go

```diff
@@ -85,21 +85,22 @@ func shellRun(v cli.Values) error {
 		EOFPrompt:         "exit",
 		HistorySearchFold: true,
 	})
-
 	if err != nil {
 		fmt.Printf("Error: %s\n", err)
 	}
 
 	defer l.Close()
 
 	home := "/"
-	// auto-discover current project with .git
-	if _, err := discoverConf([]cli.Arg{
-		{Name: _ProjectKey},
-		{Name: _ApplicationName, AllowEmpty: true},
-		{Name: _WorkflowName, AllowEmpty: true},
-	}); err == nil {
-		if r, err := repo.New("."); err == nil {
+
+	// try to discover conf for existing .git repository
+	r, err := repo.New(".")
```

Comment:
should be named differently than the err inside if

Reply:
done